utils/src/main/scala/za/co/absa/hermes/utils/SparkCompatibility.scala

```diff
@@ -21,7 +21,7 @@ import za.co.absa.commons.version.impl.SemVer20Impl.SemanticVersion
 
 object SparkCompatibility {
   val minSparkVersionIncluded: SemanticVersion = semver"2.4.2"
-  val maxSparkVersionExcluded: SemanticVersion = semver"3.2.0"
+  val maxSparkVersionExcluded: SemanticVersion = semver"3.3.0"
```

Comment:
Is there a reason to not go with 3.4 already (as it is exclusive)?

Reply:
Good idea. There is no reason as I know. Updated in commit 52bfa4945a114ffcf21a119704bd6970672c9e71.